unity-renderer/Assets/Scripts/MainScripts/DCL/Models/DCLResources/PrimitiveMeshModel.cs

```diff
@@ -21,6 +22,8 @@ public PrimitiveMeshModel(Type type)
     }
 
     public Type type;
+    public float radiusTop;
+    public float radiusBottom;
```

Comment:
Only a doubt: Does it make sense that ALL primitive meshes have a top/bottom radius? I'm thinking, for example, in a sphere... it would be a primitive shape but a top/bottom radius wouldn't have sense, no?

Reply:
I has changed it to be more general. I don't like neither of both approaches , but I think they will do the job